docfx.json

```diff
@@ -132,22 +137,17 @@
           "folder_relative_path_in_docset": "docs/standard/modern-web-apps-azure-architecture/"
         },
         "docs/architecture/modernize-with-azure-containers/": {
-     "folder_relative_path_in_docset": "docs/standard/modernize-with-azure-and-containers/"
-  },
-  "docs/architecture/serverless/": {
-     "folder_relative_path_in_docset": "docs/standard/serverless-architecture/"
-  }
-      },
-      "feedback_system": "GitHub",
-      "feedback_github_repo": "dotnet/docs",
-      "feedback_product_url": "https://aka.ms/feedback/report?space=61",
-      "featureFlags": [
-        "enable_uhf_ppe"
-      ],
-      "show_latex": true
+          "folder_relative_path_in_docset": "docs/standard/modernize-with-azure-and-containers/"
+        },
+        "docs/architecture/serverless/": {
+          "folder_relative_path_in_docset": "docs/standard/serverless-architecture/"
+        }
+      }
     },
     "fileMetadata": {
       "feedback_system": {
+        "_csharplang/**/*.md": "None",
```

Comment:
@BillWagner Did you want to turn feedback off for these folders too?

Reply:
No. I'm thinking that our docs often have more details and tutorials to learn about new features before these new features are integrated into the standard. Those recommendations could be useful.